projects/schematics/src/migrations/4_0/constructor-deprecations/constructor-deprecations.ts

```diff
@@ -130,11 +131,11 @@ export const CONSTRUCTOR_DEPRECATION_DATA: ConstructorDeprecation[] = [
   SAVED_CART_DETAILS_OVERVIEW_COMPONENT_MIGRATION_V2,
   DYNAMIC_ATTRIBUTE_SERVICE_MIGRATION,
   WINDOW_REF_MIGRATION,
-  CONFIGURATOR_CART_ENTRY_INFO_COMPONENT_MIGRATION,
```

Comment:
Why did you remove that? Please re-insert

Reply:
**CONFIGURATOR_CART_ENTRY_INFO_COMPONENT_MIGRATION** is there at another line could be found (see line 153)